app/route_rangers_api/tests/test_ingest_gtfs.py

```diff
@@ -0,0 +1,89 @@
+import pytest
```

Comment:
@mbjackson-capp if you can double check this looks good and I didn't make any wild assumptions. 

I commented a few things out that seem to no longer be needed so let me know if I am good to delete or can create a ticket to rework things later.